projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php

```diff
@@ -54,6 +55,37 @@ public function wp_ajax_videopress_get_upload_token() {
 			// 'sslverify' => false,
 		);
 
+		$endpoint = "sites/{$options['shadow_blog_id']}/media/videopress-upload-jwt";
+		$result   = Client::wpcom_json_api_request_as_blog( $endpoint, 'v2', $args, null, 'wpcom' );
+		if ( is_wp_error( $result ) ) {
+			wp_send_json_error( array( 'message' => __( 'Could not obtain a VideoPress upload JWT. Please try again later.', 'jetpack' ) ) );
+			return;
+		}
+
+		$response = json_decode( $result['body'], true );
+
+		if ( empty( $response['upload_token'] ) ) {
+			wp_send_json_error( array( 'message' => __( 'Could not obtain a VideoPress upload JWT. Please try again later.', 'jetpack' ) ) );
```

Comment:
Can this error message be different from the one above?

It makes it difficult to know which code path is failing when debugging issues if they're both the same.

Maybe something as simple as this:
```suggestion
			wp_send_json_error( array( 'message' => __( 'Could not obtain a VideoPress upload JWT. Please try again later. (empty upload token)', 'jetpack' ) ) );
```